data/.gs

```diff
@@ -1 +1 @@
-{server:'0',count:0,size:0,next:0}
```

Comment:
It is autogenerated file in JSON format, it will be regenerated without line end